src/njoy21/input/COVR/src/readCard4List.hpp

```diff
@@ -0,0 +1,25 @@
+template< typename Istream >
+std::vector< Card4 > readCard4List( Istream& is,
+ optional< std::tuple< Card2, optional< Card2z >, Card2a, Card3a > > card23a,
+ optional< std::tuple< Card2b, Card3b, Card3c > > ){
+  int ncase;
+  if( card23a != std::nullopt and card23b != std::nullopt ){
+    Log::info( "Both Card23a and Card23b exist, but only one is required." );
```

Comment:
This should be `Log::error`. You should never encounter this because of the way you wrote `readCard23x`, but it's fine to remain here. If it remains, it has to be changed to `Log::error`.